### DISCUSSION TITLE
Honeygain no longer generates income.

### DISCUSSION THREAD

**Discussion by tannguyen06** [engageub/InternetIncome#130] [Q&A]
The IPs have been active for many days, but the income is zero.
Does anyone have a solution?
![image](https://github.com/engageub/InternetIncome/assets/65990422/e8de7e1b-b148-4623-baa9-2d1705b440cc)

**Comment by engageub** [comment]
Hello,
Could you please let me know the location of these nodes being used.
I am using the same version and I see earnings. 
Could you please let me know if your proxy provider is not blocking something.
If you are using proxies from webshare, they usually block the websites. 
Also, when was the last time you downloaded the script.

Thank you

**Reply by tannguyen06** [reply]
I use residential proxies from Vietnam. For the first 7 days, I saw my earnings, but after that, there was no income at all. It seems that Honeygain has blocked my IPs or any addresses from Vietnam.

**Reply by engageub** [reply]
Hello,
You may either restart honeygain or contact honeygain support about your IP address.

Thank you

**Comment by engageub** [comment]
In addition to the above, you may also look into honeygain logs by settling ENABLE_LOGS to true in properties.conf file.

Thank you

**Reply by tannguyen06** [reply]
Thank you, I will do that and monitor further.

**Comment by superit63** [comment]
I would like to add more details related to honey gain. If your proxy is detected by DCH not Residential, you cant earn point. You can check the logs:
![image](https://github.com/engageub/InternetIncome/assets/169600953/c488b9c5-33bb-492e-9f6f-1aaef5ca290d)
If API errors with message network unusable, it means your proxy is detected by DCH.

**Reply by tannguyen06** [reply]
The logs show nothing unusual.
![image](https://github.com/engageub/InternetIncome/assets/65990422/18da362f-cc4f-4c7f-b37e-971a02e33620)

**Comment by engageub** [comment]
Hello,
Can you look into tun2socks logs as well.
Please see if there are any connections through UDP protocol. 
Also, have you made any changes related to DNS in properties.conf.
Are you using main branch or test branch. 
It may be problem with your proxies or due to low demand in that region.
Sometimes honeygain blacklists a complete subnet, when there is suspicious activity from the IPs.

Thank you

**Comment by engageub** [comment]
Hi,
Could you please paste the output of the following command from your PC.

```
sudo journalctl -u docker.service --no-pager | tail -n 100
```

Please read the recently updated [FAQ](https://github.com/engageub/InternetIncome?tab=readme-ov-file#-why-are-some-of-the-containers-for-the-same-application-offline).

Thank you
